frontend/lib/src/WidgetStateManager.ts

```diff
@@ -579,10 +581,15 @@ export class WidgetStateManager {
     })
   }
 
-  public sendUpdateWidgetsMessage(fragmentId: string | undefined): void {
+  public sendUpdateWidgetsMessage(
+    fragmentId: string | undefined,
+    isAutoRerun: boolean | undefined = undefined
```

Comment:
I set the default to `undefined` here so that we don't have to update existing tests etc. But if someone feels strongly, I can make it `boolean` only and set it to `false` and update existing tests.